src/script/lua_api/l_client.cpp

```diff
@@ -183,6 +193,10 @@ int ModApiClient::l_get_node_or_nil(lua_State *L)
 {
 	// pos
 	v3s16 pos = read_v3s16(L, 1);
+	if (getClient(L)->getCSMFlavourLimits() & CSMFlavourLimit::CSM_FL_LOOKUP_NODES) {
+		// TODO
```

Comment:
> you just break any CSM usage if you don't permit users and servers to have a consensus using configuration.

I can't see how, the usage is possible if the server is happy to enable the function..